sim/midas/src/main/cc/core/simif.h

```diff
@@ -102,6 +87,11 @@ class simif_t {
    */
   loadmem_t &get_loadmem() { return loadmem; }
 
+  /**
+   * Return a reference to the clock widget.
+   */
+  clockmodule_t &get_clock() { return clock; }
```

Comment:
I missed this in earlier review but this should be clockbridge too. 

Do you have a preference for class naming convention? I'm assuming you'd actually want this to be called `ClockModule`? Do you have some suggestions for automating more of a consistent C++ style? It might be something @sifive-benjamin-morse can help with.

Reply:
Yes, this stuff should be renamed, but i am holding it off to do alongside the metasim renaming. We should get rid of all the `*_t` names and replace it with `CamelCase` for classes. I think `clang-tidy` can help with that.